src/goose.rs

```diff
@@ -1806,7 +1829,7 @@ impl GooseUser {
         // [`test_start`](../struct.GooseAttack.html#method.test_start),
         // [`test_stop`](../struct.GooseAttack.html#method.test_stop), and during testing.
         if let Some(metrics_channel) = self.metrics_channel.clone() {
-            if let Err(e) = metrics_channel.send(GooseMetric::Request(request_metric)) {
+            if let Err(e) = metrics_channel.send(GooseMetric::Request(Box::new(request_metric))) {
```

Comment:
Unrelated change?

Reply:
Good catch! I neglected to document this:

```
% cargo clippy
    Checking goose v0.17.1-dev (devel/goose)
warning: large size difference between variants
  --> src/metrics.rs:46:1
   |
46 | / pub enum GooseMetric {
47 | |     Request(GooseRequestMetric),
   | |     --------------------------- the largest variant contains at least 280 bytes
48 | |     Transaction(TransactionMetric),
   | |     ------------------------------ the second-largest variant contains at least 72 bytes
49 | |     Scenario(ScenarioMetric),
50 | | }
   | |_^ the entire enum is at least 280 bytes
   |
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#large_enum_variant
   = note: `#[warn(clippy::large_enum_variant)]` on by default
help: consider boxing the large fields to reduce the total size of the enum
   |
47 |     Request(Box<GooseRequestMetric>),
   |             ~~~~~~~~~~~~~~~~~~~~~~~

warning: `goose` (lib) generated 1 warning
    Finished dev [unoptimized + debuginfo] target(s) in 1.92s
```